FF1Lib/FF1Rom.cs

```diff
@@ -303,6 +308,7 @@ public static string EncodeFlagsText(Flags flags)
 			bits[27] = flags.TeamSteak;
 			bits[28] = flags.Music == MusicShuffle.Standard;
 			bits[29] = flags.Music == MusicShuffle.Nonsensical;
+			bits[30] = flags.ShuffleLeader;

```

Comment:
This will need to be `bits[28]` because of the (sad) JS code on the website.  And music will need to be 29 and 30.